.github/workflows/tests.yml

```diff
@@ -145,7 +145,7 @@ jobs:
         uses: "shivammathur/setup-php@v2"
         with:
           coverage: "none"
-          php-version: "8.1"
+          php-version: "8.3"
```

Comment:
should we run the level tests in all php versions? or maybe lowest and highest supported version?

Reply:
No it's fine to run it just once.

Reply:
E.g. there's nothing that changes for levels on different PHP versions. If some rule changes behaviour based on PHP version we test in that rule's test.